src/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
`.prettierrc.json` file should be placed in project's root directory (where the `package.json` resides`